raft/test/raft_test.cpp

```diff
@@ -575,8 +574,8 @@ namespace bzn
 
         resp.clear();
 
-        // send append entry with commit index of 1... and follower commits and updates its match index
```

Comment:
Because these are all sent with the same prev_log_entry and prev_log_term (both 0), they should all overwrite each other.